app/views/exchanges/scheduled_events/show.html.erb

```diff
@@ -22,4 +22,4 @@
 
 <%= link_to 'Back', exchanges_scheduled_events_path, class: "btn btn-default" %>
 <%= link_to "Edit", edit_exchanges_scheduled_event_path(@scheduled_event), class: "btn btn-default" %>
-<%= link_to "Delete This Occurence", delete_current_event_exchanges_scheduled_events_path(id: @scheduled_event.id, time: @time), class: "btn btn-danger", html: { method: :post } %>
+<%= link_to "Delete This Occurence", delete_current_event_exchanges_scheduled_events_path(id: @scheduled_event.id, time: @time), class: "btn btn-danger", html: { method: :get } %>
```

Comment:
basically we are not deleting anything here we are creating a data in eventexception table and hiding the event from calender, we are not doing delete action here, just creating data in event exception table and hiding data from calender